src/main/java/uk/gov/hmcts/probate/controller/DataExtractController.java

```diff
@@ -10,6 +10,7 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.probate.service.DormantCaseService;
```

Comment:
will there be a new FunctionalTest on this class - especially since we are rearranging the scheduling mechanism?

Reply:
Not required